### ISSUE TITLE
switching jobs should remember equipment worn

### ISSUE THREAD

**Issue by kaincenteno** [project-topaz/topaz#504]
**_I have:_**

- [x] searched existing issues (http://project-topaz.com/issues/) to see if the issue has already been opened
- [x] checked the commit log to see if the issue has been resolved since my server was last updated

**_Additional Information_** (Steps to reproduce/Expected behavior) **:** 
When switching jobs, it should remember the clothes the pc was wearing. Then when switching back it should auto equip them. Currently PC is nude when switching jobs

**Comment by TeoTwawki** [comment]
this is true as long as the item remains in the same container as when you were last on that job: move it into/out of a wardrobe and when you change job the item won't be re-equipped.

But otherwise, you should get re-geared automatic when you change back to a job.
